h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -1227,14 +1229,50 @@ protected Frame rebalance(final Frame original_fr, boolean local, final String n
     return rebalanced_fr;
   }
 
+  private double rebalanceRatio() {
+    String mode = H2O.getCloudSize() == 1 ? "single" : "multi";
+    String ratioStr = System.getProperty(H2O.OptArgs.SYSTEM_PROP_PREFIX + "rebalance.ratio." + mode, "1.0");
+    return Double.parseDouble(ratioStr);
```

Comment:
Do you need to care about handling non-parsable numbers here? Or just let it fail and report error via MB?